### ISSUE TITLE
Can't rebuild message files

### ISSUE THREAD

**Issue by rachel-fenichel** [google/blockly#1934]
### Problem statement
I can't rebuild message files with `build.py` because `js_to_json` is failing.  It works on master, but not on develop.

### Expected Behavior
Running `build.py` rebuilds all langfiles and outputs appropriate JSON for each language.

### Actual Behavior
`build.py` calls `18n/js_to_json.py` to go from `messages.js` to `en.json`, `qqq.json`, etc.

`msg/json/en.json` is essentially empty after this command:
`./i18n/js_to_json.py --input_file msg/messages.js --output_dir msg/json/`

Reverting the change to `js_to_json` in this [commit](https://github.com/google/blockly/commit/4cb4b42f8d7eb7418d3dd297b543e5e0ac4f27b0#diff-3e0a4a136f04ff1a896df8acbf0a14eb) solves the problem for me.

That is, changing the input def pattern from
` re.compile("""Blockly.Msg\[(\w*)\]\s*=\s*'(.*)';?\r?$""")`
to
`re.compile("""Blockly.Msg.(\w*)\s*=\s*'(.*)';?\r?$""")`
makes it work again.

### Steps to Reproduce
Update from develop
Run `./build.py`.

### Stack Traces

```
Error reading ./msg/json/en.json
Traceback (most recent call last):
  File "i18n/create_messages.py", line 166, in <module>
    main()
  File "i18n/create_messages.py", line 78, in main
    source_defs = read_json_file(os.path.join(os.curdir, args.source_lang_file))
  File "/home/fenichel/blockly/gblockly-rr/i18n/common.py", line 64, in read_json_file
    raise InputError(filename, str(e))
common.InputError: ./msg/json/en.json: Expecting property name enclosed in double quotes: line 9 column 1 (char 176)
('Error running i18n/create_messages.py: ', CalledProcessError())
```

### Operating System and Browser
I wonder if this is OS-specific, because I expect that @AnmAtAnm and @NeilFraser tested rebuilding when they were debugging message files.

I'm on Ubuntu.

**Comment by rachel-fenichel** [comment]
This is a blocking bug for the June release.

**Comment by rachel-fenichel** [comment]
Fixed by #1935.
